stories/atoms/SuggestionItem.stories.tsx

```diff
@@ -12,20 +12,30 @@ type StorySuggestionItem = StoryObj<typeof SuggestionItem>;
 export const _SuggestionItem: StorySuggestionItem = {
```

Comment:
I would add 2 stories, one for single select and one for multi select for clarity, and showing the callbacks correctly.

Also if you rebase on `master` there should be some stories now for the SelectedItemBadge. Maybe that can be also enriched with using this components, so that it can be fully tested/demoed.